### ISSUE TITLE
Finite-Time Gradient Descent-Based Adaptive Neural Network Finite-Time Control Design for Attitude Tracking of a 3-DOF Helicopter. (arXiv:2107.12924v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#9340]
Finite-Time Gradient Descent-Based Adaptive Neural Network Finite-Time Control Design for Attitude Tracking of a 3-DOF Helicopter. (arXiv:2107.12924v1 [eess.SY])<br>
https://ift.tt/3iT7PGm<br>
<p>This paper investigates a novel finite-time gradient descent-based adaptive neural network finite-time control strategy for the attitude tracking of a 3-DOF lab helicopter platform subject to composite disturbances. First, the radial basis function neural network (RBFNN) is applied to estimate lumped disturbances, where the weights, centers and widths of the RBFNN are trained online via finite-time gradient descent algorithm. Then, a finite-time backstepping control scheme is constructed to fulfill the tracking control of the elevation and pitch angles. In addition, a hybrid finite-time differentiator (HFTD) is introduced for approximating the intermediate control signal and its derivative to avoid the problem of "explosion of complexity" in the traditional backstepping design protocol. Moreover, the errors caused by the HFTD can be attenuated by the combination of compensation signals. With the aid of the stability theorem, it is proved that the closed-loop system is semi-globally uniformly ultimately boundedness in finite time. Finally, a comparison result is provided to illustrate the effectiveness and advantages of the designed control strategy.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
